### DISCUSSION TITLE
Please accept my plugin

### DISCUSSION THREAD

**Discussion by stephrobert** [asdf-community/.github#28] [General]
Hi,
I wrote a plugin for Regula at https://github.com/stephrobert/asdf-regula

I would like to move the new plugin here please:

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
